guide/english/css/css3-media-queries/index.md

```diff
@@ -37,7 +33,7 @@ The `p` tag will have a padding of 30px as soon as the screen reaches min 768px
 }
 ```
 
-The `p` tag will have a padding of 30px as soon as the screen reaches min 768px height and its orientation is landscape.
+`p` tags will have 30px of padding when the screen width is at least 768px AND its orientation is landscape.
```

Comment:
It should be `height` instead of `width` here.
```suggestion
`p` tags will have 30px of padding when the screen height is at least 768px AND its orientation is landscape.
```
or you can change the media query to refer to `min-width` instead of `min-height`.